.github/workflows/deploy-services-sandbox.yml

```
@@ -0,0 +1,81 @@
+name: Deploy Service's Test Sandbox Stack to AWS
+
+on:
+  push:
+    branches:
+      - services-hyp3-sandbox
```

Naming the branch after the deployment might help us stay more organized.
```suggestion
      - hyp3-services-test
```